resident/resident-service/src/main/java/io/mosip/resident/util/Utilities.java

```diff
@@ -1,341 +1,364 @@
+import org.json.simple.parser.JSONParser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.*;
+
+/**
+ * The Class Utilities.
+ *
+ * @author Girish Yarru
+ */
+@Component
+
+/**
+ * Instantiates a new utilities.
+ */
+@Data
+public class Utilities {
+
+    private final Logger logger = LoggerConfiguration.logConfig(Utilities.class);
+    /**
+     * The reg proc logger.
+     */
+    private static final String sourceStr = "source";
+
+    /**
+     * The Constant UIN.
+     */
+    private static final String UIN = "UIN";
+
+    /**
+     * The Constant FILE_SEPARATOR.
+     */
+    public static final String FILE_SEPARATOR = "\\";
+
+    /**
+     * The Constant RE_PROCESSING.
+     */
+    private static final String RE_PROCESSING = "re-processing";
+
+    /**
+     * The Constant HANDLER.
+     */
+    private static final String HANDLER = "handler";
+
+    /**
+     * The Constant NEW_PACKET.
+     */
+    private static final String NEW_PACKET = "New-packet";
+
+    @Value("${IDSchema.Version}")
+    private String idschemaVersion;
+
+    @Value("${provider.packetwriter.resident}")
+    private String provider;
+
+    @Autowired
+    @Qualifier("restTemplate")
+    private RestTemplate residentRestTemplate;
+
+    @Autowired
+    private ObjectMapper objMapper;
+
+    @Autowired
+    private Environment env;
+
+    @Autowired
+    private ResidentServiceRestClient residentServiceRestClient;
+
+    @Autowired
+    private TokenGenerator tokenGenerator;
+
+    /**
+     * The config server file storage URL.
+     */
+    @Value("${config.server.file.storage.uri}")
+    private String configServerFileStorageURL;
+
+    /**
+     * The get reg processor identity json.
+     */
+    @Value("${resident.identityjson}")
+    private String getRegProcessorIdentityJson;
+
+    /**
+     * The id repo update.
+     */
+    @Value("${id.repo.update}")
+    private String idRepoUpdate;
+
+    /**
+     * The vid version.
+     */
+    @Value("${resident.vid.version}")
+    private String vidVersion;
+
+
+    /**
+     * The Constant NAME.
+     */
+    private static final String NAME = "name";
+
+    private static final String VALUE = "value";
+
+    private String mappingJsonString = null;
+
+    private static String regProcessorIdentityJson = "";
+
+    @PostConstruct
+    private void loadRegProcessorIdentityJson() {
+        regProcessorIdentityJson = residentRestTemplate.getForObject(configServerFileStorageURL + getRegProcessorIdentityJson, String.class);
```

Comment:
we changed config to residentIdentityJson. Please change name of variable also.

Reply:
The name of the variable is not changed in the sandbox config. @MonobikashDas 

resident.identityjson=registration-processor-identity.json